src/Microsoft.TestPlatform.Common/Utilities/RunSettingsUtilities.cs

```diff
@@ -190,4 +190,14 @@ public static IEnumerable<string> GetTestAdaptersPaths(string runSettings)
         return testAdaptersPaths;
     }
 
+    /// <summary>
+    /// Gets the test adapter loading strategy
+    /// </summary>
+    /// <param name="runSettings">Test run settings</param>
+    /// <returns>Test adapter loading strategy</returns>
+    internal static TestAdapterLoadingStrategy GetLoadingStrategy(string runSettings) {
+        var runConfiguration = XmlRunSettingsUtilities.GetRunConfigurationNode(runSettings);
+        
+        return runConfiguration?.TestAdapterLoadingStrategy ?? TestAdapterLoadingStrategy.Default;
```

Comment:
I see nullcheck here but not in other place where we use `XmlRunSettingsUtilities.GetRunConfigurationNode(runSettings);` is it ok?

Reply:
Here, it might be null, in other places we always have one.

Reply:
It's not required here too. Fixed.